docs/root/configuration/http/http_filters/rate_limit_quota_filter.rst

```diff
@@ -40,12 +40,25 @@ has expired, the filter can be configured to continue using the last quota assig
 The rate limit quota filter reports the request load for each bucket to the RLQS with the configured ``reporting_interval``. The RLQS may rebalance quota assignments based on the request
 load that each Envoy receives and push new quota assignments to Envoy instances.
 
-When the connection to RLQS server fails, the filter will fall back to either the
+Failure modes
+^^^^^^^^^^^^^
+
+In case the connection to RLQS server fails, the filter will fall back to either the
 :ref:`no assignment behavior <envoy_v3_api_field_extensions.filters.http.rate_limit_quota.v3.RateLimitQuotaBucketSettings.no_assignment_behavior>`
 if it has not yet received a rate limit quota or to the
 :ref:`expired assignment behavior <envoy_v3_api_field_extensions.filters.http.rate_limit_quota.v3.RateLimitQuotaBucketSettings.expired_assignment_behavior>` if
 connection could not be re-established by the time the existing quota expired.
 
+In case RLQS client, for any reason (including RLQS server connection failures), doesn't receive
```

Comment:
Suggest writing this sentence as:

"""
In case the RLQS client doesn't receive the initial bucket assignment (for any reason, including RLQS server connection failures) within predetermined [2]_ time, such buckets will eventually be purged from memory.
"""

Reply:
Fixed in https://github.com/envoyproxy/envoy/pull/30581/commits/e9abcc9b639089fd3e79e160359e96061972614e.